system_health.py

```
@@ -0,0 +1,160 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)